### PR TITLE
Make any Retired vault visible if the deposited amount is non-zero.

### PR DESCRIPTION
**Pull request by JordanRaphael**
It's been observed that many users are panicking when they can't find their vault, which is the case for Retired vaults since they are hidden, even when the deposited amount is non-zero. Right now, if someone wants to find these vaults he has to click the "Retired" checkbox. 
I think by making Retired vaults visible to wallets with deposited amount greater than 0 will release some stress from the users. Also, I believe that placing these vaults at the top of the list is a good idea, since they are Retired, and the funds should be removed. If you could point me to the right direction, I might be able to do that as well in a later update.

Edit: because I don't have a deposited amount in any Retired vault I couldn't properly test it, I tested it by changing the expression so to work with 0 balances as well.

### CODE REVIEW

File: src/features/vault/hooks/useFilteredPools.js [hunk 1]
```diff
@@ -67,9 +67,9 @@ function showBoosted(pools) {
   });
 }
 
-function hideDecomissioned(pools) {
+function hideDecomissioned(pools, tokens) {
   return pools.filter(pool => {
-    return pool.status !== 'eol' && pool.status !== 'refund';
+    return (pool.status !== 'eol' && pool.status !== 'refund') || (tokens[pool.earnedToken] && tokens[pool.earnedToken].tokenBalance > 0);
```
Comment (nienow) [inline src/features/vault/hooks/useFilteredPools.js:72]: Should this be checking the `token` instead of the `earnedToken`?

Comment (nienow) [reply src/features/vault/hooks/useFilteredPools.js:72]: Oh, sorry, I was looking at the wrong code - was looking at the boost pools. Everything looks good to me, but I can't test either.

**Comment by roman-monk** [comment]
i've tested by setting 'eol' status to the vaults in bsc_pools where i have mooTokens 😄

**Comment by roman-monk** [comment]
@JordanRaphael please DM me in discord for a bounty

**Comment by JordanRaphael** [comment]
@roman-monk 9520

**Pull request merged**